OMCompiler/Compiler/SimCode/SimCodeVar.mo

```diff
@@ -104,6 +104,7 @@ public uniontype SimVar "Information about a variable in a Modelica model."
     Option<String> matrixName "if the varibale is a jacobian var, this is the corresponding matrix";
     Option<Variability> variability "FMI-2.0 variabilty attribute";
     Option<Initial> initial_ "FMI-2.0 initial attribute";
+    Boolean isCseVariable "check if variable is a commonSubExpression";
```

Comment:
This PR shouldn't just be about cse variables, but about all internal variables. So I propose to change the name of the attribute to something like `Boolean isInternalVariable`.

Reply:
i can change it